data-explorer/kusto/concepts/data-soft-delete.md

```diff
@@ -110,3 +111,16 @@ To delete all the records that contain data of a given user:
 ### Output
 
 The output of the command contains information about which extents were replaced.
+
+## Soft delete for materialized views
+
+Soft deleted can be executed also on materialized views, where the same concepts explained above are applied similarly for materialized views.
+
+### Syntax
+
+`.delete` [`async`] `materialized-view` *MaterializedViewName* `records` [`with (`propertyName `=` propertyValue [`,` ...]`)`] `<|` *Predicate*
+
+> [!NOTE]
+> The same restrictions on the *Predicate* mentioned above apply here as well.
+> Soft delete is expected to fail in case a materialization of the view is being executed concurrently (specifically, in case there is an intersection between the extents those two commands are executing on).
+> Usage of function 'materialized_view' is not allowed in *Predicate*.
```

Comment:
`materialized_view()` should link to the page of the function and should use `` (not '').

Reply:
If I'm already adding a link, I think it's ok to remove the apostrophe